queries.sql

```diff
@@ -36,37 +36,41 @@ SELECT * FROM albums WHERE title LIKE 'A%' ORDER BY title;
 -- Remember: run ".schema invoices" to see the structure of the "invoices" table
 
 -- Every invoice
-
+SELECT * FROM invoices;
 -- Every invoice ordered by total invoice amount ("total")
-
+SELECT * FROM invoices ORDER BY total;
 -- Every invoice with a total greater than 10
-
+SELECT * FROM invoices WHERE total > 10;
 -- The 10 least expensive invoices
+SELECT * FROM invoices ORDER BY total LIMIT 10;
 -- Remember: ORDER BY orders from lowest-to-highest by default
 SELECT * FROM invoices ORDER BY total LIMIT 10;
 
 -- The 10 most expensive invoices
-
+SELECT * FROM invoices ORDER BY total DESC LIMIT 10;
 -- The 15 most recent invoices
-
+select * FROM invoices ORDER BY invoice_date DESC LIMIT 15;
 -- The 15 oldest invoices
-
+select * FROM invoices ORDER BY invoice_data LIMIT 15;
 -- The 10 most expensive invoices from the US
-
+select * FROM invoices WHERE billing_country = "USA" ORDER BY total DESC LIMIT 10;
 -- The 10 least expensive invoices from the US
-
+select * from invoices where billing_country = "USA" order by total LIMIT 10;
 -- The 10 most expensive invoices from outside the US
+select * from invoices where billing_country != "USA" ORDER BY total DESC LIMIT 10;
 -- Hint: If "=" means equal, use "!=" to mean "not equal"
 
 -- Every invoice from Chicago, IL
 SELECT * FROM invoices WHERE billing_city = "Chicago" AND billing_state = "IL" AND billing_country="USA";
 
 -- A list of all the invoices worth more than $5.00 from Chicago, IL
+SELECT * FROM invoices WHERE billing_city = "Chicago" AND billing_state = "IL" AND billing_country = "USA" AND total > 5;
 
 -- The billing addresses of the 5 most valuable invoices from Mountain View CA
 -- Gotta reward those big spenders!
-
+SELECT billing_address FROM invoices WHERE billing_city = "Mountain View" AND billing_state = "CA" AND billing_country = "USA" ORDER BY total DESC LIMIT 5;
 -- A list of the 10 most valuable invoices made before January 1st, 2010
+SELECT * from invoices WHERE invoice_date < 2010-01-01 ORDER BY total DESC LIMIT 10;
```

Comment:
Make sure you get SQLite running on your computer so you can test out these queries. The README has instructions. You can use home brew to install it on a Mac.

(This query won't quite work.)